### PR TITLE
[onDeviceUI] Unifying React Native template.

### PR DESCRIPTION
**Pull request by Gongreg**
Part of #3903 

Currently there are two templates for React Native (vanilla and crna). Since we are 
unifying RN Storybook experience it makes sense to also unify Storybook installation.

### CODE REVIEW

**Review by igor-dv — APPROVED** [review]
LGTM. I would mark it as breaking change though

**Comment by Gongreg** [comment]
@igor-dv This one will impact new users only so I am not sure it counts as a breaking change.

File: lib/cli/lib/initiate.js [hunk 1]
```diff
@@ -96,26 +95,20 @@ export default function(options, pkg) {
...
-          .then(commandLog('Adding storybook support to your "Create React Native App" app'))
+      case types.REACT_NATIVE: {
+        return reactNativeGenerator(npmOptions)
+          .then(commandLog('Adding storybook support to your "React Native" app'))
           .then(end)
           .then(() => {
-            logger.log(chalk.red('NOTE: CRNA app installation is not 100% automated.'));
-            logger.log(`To quickly run storybook, replace contents of ${app} with:\n`);
+            logger.log(chalk.red('NOTE: installation is not 100% automated.'));
+            logger.log(`To quickly run storybook, replace contents of your app entry with:\n`);
```
Comment (Gongreg) [inline lib/cli/lib/initiate.js]: Add link to manual setup readme

**Review by ndelangen — APPROVED** [review]

**Comment by Gongreg** [comment]
Lets merge this together when merging #3903.

**Pull request merged**